content-cache/tests/integration/conftest.py

```diff
@@ -47,37 +49,61 @@ def charm_file_fixture(pytestconfig: pytest.Config) -> str:
     return f"./{file}"
 
 
+@pytest_asyncio.fixture(name="config_charm_file", scope="module")
+async def config_charm_file_fixture(ops_test: OpsTest) -> AsyncIterator[str]:
```

Comment:
Hmm, something is weird, this doesn't have a docstring, why doesn't our CI complain?

Reply:
I will look into this. Something might be wrong with the workflows.

Reply:
The issue is with the `pyproject.yaml` ignore some lints for files under `tests/*`.
I think `D100,D103` should not be ignored.
This is a issue with other repos as well such as https://github.com/canonical/github-runner-image-builder-operator/blob/704f3755d624e77b09af9e340a3fbebe7b5c7c32/pyproject.toml#L41.
I do know the scope of the issue across repos yet, but I think more repo are affect.

Reply:
I updated the lint setting in pyproject and fix the docstrings.